frontend/src/tabs/zoom.pug

```diff
@@ -14,14 +14,15 @@
         v-bind:title="'Click to open the repo'"
       )
         span {{ filteredUser.repoName }}
-    .zoom__title--chart.medium-font(v-if="!info.zIsMerge")
+    .author.medium-font(v-if="!info.zIsMerge")
       span &#8627; &nbsp;
       span(v-if="info.zFilterGroup === 'groupByAuthors'") {{ filteredUser.repoName }}
       span(v-else) {{ filteredUser.displayName }} ({{ filteredUser.name }})
-    .zoom__title--period.medium-font
+    .period.medium-font
       span &#8627; &nbsp;
       span {{ info.zSince }} to {{ info.zUntil }} &nbsp;
       a(v-on:click="openSummary") [Show ramp chart for this period]
+  .zoom__title
```

Comment:
what's the reason behind adding this?

Reply:
This is present in the original CSS. So I have kept it as it is, and just removed `group-name`, `author` and `period` from it.